profiles/common/modules/features/nexteuropa_trackedchanges/NETCInfo.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @file
+ * Contains \NextEuropaTrackedChangesInfo.
+ */
+
+/**
+ * Class NETCInfo.
+ */
+class NETCInfo extends Entity {
```

Comment:
We could maybe move this under `./src` and use namespacing. In order to do so perform the following steps:

1. add `dependencies[] = registry_autoload` to `nexteuropa_trackedchanges.info` 
2. add `registry_autoload[] = PSR-4` to `nexteuropa_trackedchanges.info`
3. add `namespace Drupal\nexteuropa_trackedchanges;` on both `.inc` files
4. rename extension from `.inc` to `.php`
5. Move files under `./src`
6. Replace all occurrences of `NETC*` with `Drupal\nexteuropa_trackedchanges\NETC*` in info hooks